### DISCUSSION TITLE
Library view alternatives

### DISCUSSION THREAD

**Discussion by Omnilocus** [rern/rAudio#1142] [Feature Requests]
Would it be possible to make a folder-view alternative to library? 
(and IGNORE cue/tags)
(similar to Sqeeze/Logitec...)

**Comment by Omnilocus** [comment]
Or alternative way of sorting artist/folder/...?
My problem is finding artist (files) that is not tagged, or incorrectly tagged...

**Reply by ceejayemm** [reply]
In the Library, click on NAS and then on the name given to the NAS library share (in my case NAS_Music). This will list all the folders in the source library folder by Artist. Contents of there folders (in my case Albums) can be similarly viewed by clicking on the folder name.  Clicking then on the individual track name gives access to the Tag Editor to add/ fix any additional tags - also available is 'Add to Playlist, Add Similar etc).  

The trick in all of the above is to click on the item name not the the attached icon.

Regards.

Chris
